packages/angular/cli/lib/cli/index.ts

```diff
@@ -48,10 +48,30 @@ export default async function(options: { testing?: boolean, cliArgs: string[] })
     return 0;
   } catch (err) {
     if (err instanceof Error) {
-      logger.fatal(err.message);
-      if (err.stack) {
-        logger.fatal(err.stack);
+      try {
+        const fs = await import('fs');
+        const os = await import('os');
+        const tempDirectory = fs.mkdtempSync(fs.realpathSync(os.tmpdir()) + '/' + 'ng-');
+        const logPath = tempDirectory + '/angular-debug.log';
```

Comment:
Any reason why the file name is named `-debug` Instead of `-errors`?

Reply:
not particularly; changed it to `errors`.